src/app.js

```diff
@@ -1,21 +1,27 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  const { url } = req;
+  const correctPath = url === ('/file' || '/file/')
+    ? 'public/index.html'
+    : url.substring(1)
+      .replace('file', 'public');
+
+  if (!correctPath.startsWith('/file')) {
```

Comment:
In the previous code you remove `/` and save results to `correctPath`, so it will never start from that character. Maybe you wanted to check `url`?